vars/RedHat-7.yml

```diff
@@ -1,9 +1,9 @@
 ---
 __postgresql_version: "9.2"
-__postgresql_data_dir: "/var/lib/pgsql/data"
+__postgresql_data_dir: "/var/lib/pgsql"
```

Comment:
`/data` removed here (and not added later it seems) seems like a non downward-compatible change